### ISSUE TITLE
system is not working

### ISSUE THREAD

**Issue by arademaker** [delph-in/matrix#674]
https://matrix.ling.washington.edu/customize/matrix.cgi 

is not working. How to make sure from the web interface if it is the last version in the `trunk` branch?

**Comment by emilymbender** [comment]
We're looking into it --- hopefully can be addressed during Seattle business hours today, otherwise early this week.

The time stamp on the Matrix web page should allow you to compare to trunk. The main page doesn't have the head revision right now. This variant is currently closer, at least (and not down):

https://matrix.ling.washington.edu/ltxom-test/matrix.cgi

**Comment by arademaker** [comment]
So the last commit in the trunk was `Sep 29th 2022`

https://github.com/delph-in/matrix/commits/trunk

But https://matrix.ling.washington.edu/ltxom-test/matrix.cgi says `28th Jul 2022`. Am I looking in the right place?

**Comment by arademaker** [comment]
Hi @emilymbender, this issue may need to be more specific... but I hope reporting here would be fine. 

@leoalenc has been using the matrix from [1] and you suggested the link above (listed as [2] below):

1. https://matrix.ling.washington.edu/trunktest/matrix.cgi
2. https://matrix.ling.washington.edu/ltxom-test/matrix.cgi#

It turns out that [1] can load our [choices file](https://github.com/LR-POR/PorGram/blob/main/choices) but [2] can't load it. Any idea? I think this problem was reported somewhere, but I could not find it or the solution. It looks like the new version of the matrix require some manual modification of the choices file, right? 

The. correct URL for using the last stable version of the matrix is [2], right? The `trunktest` in the url suggest the opposite.

**Comment by emilymbender** [comment]
[2] isn't necessarily the last stable version --- just more updated than [1]. We plan to update [1] after my class is done using it intensively, so in a couple of weeks.

I'm surprised that [2] doesn't accept your choices file, while [1] does. Could you email it to us?

**Comment by arademaker** [comment]
In my last comment above I provide the link to our choices file. I will mail it to you too. 

Thank you

**Comment by emilymbender** [comment]
Much belatedly, I've made a start at looking into this. From the error trace, it looks like it's the validation system that is choking on something about the lexicon file --- specifically about comparing the features between supertypes & subtypes somewhere in the lexicon file.

We should definitely track down whether this persists in the most current version of the Matrix. @KerenR3 and/or @taraw28 if you get a chance can you try uploading the file linked from @arademaker 's message from Feb 7 into the latest?

@arademaker --- is this still in your way?

**Comment by KerenR3** [comment]
I ran the latest version of the trunk branch (52feedd4) on a local web server and I can upload the file, and I get a validation error (several red asterisks).

**Comment by arademaker** [comment]
Yes, It would be great to have updates about it. We keep in our repo the last choice file used to produce the files in the repo... https://github.com/LR-POR/PorGram

**Comment by emilymbender** [comment]
Thanks. It seems like this should be somewhat better next time we update the live site. Meanwhile, if you want to see what the issue is with the choices file, I recommend running the trunk version of validate.py over it.
